Description: Fix show up window manager in display manager.
Author: Mateusz Łukasik <mati75@linuxmint.pl>
Bug-Debian: http://bugs.debian.org/596427
Bug-Debian: http://bugs.debian.org/703986
Forwarded: not-needed

--- a/evilwm.desktop
+++ b/evilwm.desktop
@@ -2,7 +2,7 @@
 Type=Application
 Name=evilwm
 Exec=evilwm
-NoDisplay=true
+NoDisplay=false
 X-GNOME-WMName=evilwm
 # evilwm doesn't talk XSMP yet, so don't start in the WindowManager phase:
 X-GNOME-Autostart-Phase=Applications
